src/components/Search/engines/StopFinder.js

```diff
@@ -3,13 +3,19 @@ import React from 'react';
 import Engine from './Engine';
 
 class StopFinder extends Engine {
-  constructor(endpoint = 'https://api.geops.io/stops/v1/') {
+  constructor(endpoint = 'https://api.geops.io/stops/v1/', options) {
     super();
+    this.params = options || {};
     this.endpoint = endpoint;
   }
 
   search(value) {
-    return fetch(`${this.endpoint}?&q=${value}&key=${this.apiKey}`)
+    const paramString = `${Object.keys(this.params)
```

Comment:
I would prefer to extract valid API parameters from `options` explicitly to make it easier to understand whats going on.

Reply:
Agreed, thanks for pointing that out